web/client/plugins/GeoStoryEditor.jsx

```diff
@@ -24,17 +24,22 @@ const GeoStoryEditor = ({
     mode = Modes.VIEW,
     setEditingMode = () => {},
     story = {}
-}) => (<div
+}) => (mode === Modes.EDIT ? <div
     key="left-column"
     className="ms-geostory-editor"
     style={{ order: -1, width: 400, position: 'relative' }}>
     <Builder
+        scrollTo={(id, options = { behavior: "smooth" }) => {
+            const element = document.getElementById(id);
```

Comment:
Is this the same of the one in utils? maybe you cna reuse it and changeOptions overriding them